packages/ember-simple-auth/addon/authenticators/devise.js

```diff
@@ -1,11 +1,11 @@
 import { Promise } from 'rsvp';
 import { isEmpty } from '@ember/utils';
 import { run } from '@ember/runloop';
-import { merge, assign as emberAssign } from '@ember/polyfills';
+import { merge } from '@ember/polyfills';
```

Comment:
Isn't `merge` the one that was [removed from `@ember/polyfills` in 4.0](https://api.emberjs.com/ember/4.0/modules/@ember%2Fpolyfills)?